engine/runtime/src/test/java/org/enso/interpreter/test/FindExceptionMessageTest.java

```diff
@@ -67,6 +70,71 @@ public void testThrowNPEWithName() {
     }
   }
 
+  @Test
+  public void errorThrowDeep() {
+    var src =
+        """
+    from Standard.Base import all
+    import Standard.Base.Errors.Illegal_Argument.Illegal_Argument
+
+    deep n = if n <= 0 then Error.throw (Illegal_Argument.Error "Problem"+n.to_text) else deep n-1
+
+    main =
+        d = deep 10
+        d
+    """;
+
+    var res = evalModule(ctx, src);
+    assertTrue("Expecting error: " + res, res.isException());
+    assertEquals("Standard.Base.Error.Error", res.getMetaObject().getMetaQualifiedName());
+
+    try {
+      throw res.throwException();
+    } catch (PolyglotException ex) {
+      assertNotNull("Has source location", ex.getSourceLocation());
+      var throwCode = ex.getSourceLocation().getCharacters().toString();
+      assertNotEquals("Throw code found in the source code: " + throwCode, src.indexOf(throwCode));
```

Comment:
This assert looks suspicious.

It seems to essentially do
```
assert ("Throw code found in the source code: " + throwCode) != src.indexOf(throwCode);
```
This seems to be trivially true. Aren't you missing one more argument in that assert?

Reply:
Thank you: 80dc1cb19a